sdk/translation/azure-ai-translation-document/samples/async_samples/sample_list_document_statuses_with_filters_async.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_list_document_statuses_in_a_translation_job_with_filters_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to list all the docs in some translation job for the resource 
+    using different kind of filters/sorting/paging options
+
+    To set up your containers for translation and generate SAS tokens to your containers (or files)
+    with the appropriate permissions, see the README.
+
+USAGE:
+    python sample_list_document_statuses_in_a_translation_job_with_filters_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_DOCUMENT_TRANSLATION_ENDPOINT - the endpoint to your Document Translation resource.
+    2) AZURE_DOCUMENT_TRANSLATION_KEY - your Document Translation API key.
+"""
+
+import os
+import asyncio
+
+async def sample_list_document_statuses_in_a_translation_job_with_filters_async(self, client):
+    # import libraries
+    from azure.core.credentials import AzureKeyCredential
+    from azure.ai.translation.document import (
+        DocumentTranslationClient,
+    )
+    from datetime import datetime
+    import uuid
+
+    # obtain client secrets
+    endpoint = os.environ["AZURE_DOCUMENT_TRANSLATION_ENDPOINT"]
+    key = os.environ["AZURE_DOCUMENT_TRANSLATION_KEY"]
+    job_id = os.environ["JOB_ID"]  # this should be the id for the job you'd like to list docs for!
+
+    # authorize client
+    client = DocumentTranslationClient(endpoint, AzureKeyCredential(key))
+
+    # set your filters
```

Comment:
I think we can do this for now, but a few things:

1) We need to add more details to the comment here describing how these are just sample values for the filters and to please comment/uncomment/change to what you are interested in using.
2) Not for this sprint, but eventually we will enable these samples to be test run in the CI pipelines. At that point, we may need to convert all of these filters to environment variables to ensure we get some documents returned.